addons/mail/static/src/models/guest.js

```diff
@@ -30,6 +30,13 @@ registerModel({
         _computeAvatarUrl() {
             return `/web/image/mail.guest/${this.id}/avatar_128?unique=${this.name}`;
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsOnline() {
+            return ['online', 'away'].includes(this.im_status);
+        },
```

Comment:
Note that this is duplicate with `Partner` code.
It's necessary because you don't change much the template of `PartnerImStatusComponent`.

Not a big deal for this PR, but would be nice to have model `Person` that covers concerns that are the same between the models, such as the `name` or the `im_status`.